causal_testing/json_front/json_class.py

```diff
@@ -89,10 +89,11 @@ def setup(self):
         self._populate_metas()
 
     def _create_abstract_test_case(self, test, mutates, effects):
+
         abstract_test = AbstractCausalTestCase(
             scenario=self.modelling_scenario,
             intervention_constraints=[mutates[v](k) for k, v in test["mutations"].items()],
-            treatment_variables={self.modelling_scenario.variables[v] for v in test["mutations"]},
+            treatment_variables=self.modelling_scenario.variables[next(iter(test["mutations"]))],
```

Comment:
What's this line doing? Could we not just do `self.modelling_scenario.variables[test["mutations"][0]]`?

Reply:
test["mutations"] returns a dictionary with a single key:value pair in it. Because it's unordered `[0]` doesn't work so the extra `next(iter( ))` is the cleanest way I can think of. I'm not a big fan of it either however, I'll also think if there's other ways!

Reply:
Thinking about it, we could probably insist on a single mutation which is the treatment variable. I made it a dict when we were going to support multiple mutations.

Reply:
Next PR will address this